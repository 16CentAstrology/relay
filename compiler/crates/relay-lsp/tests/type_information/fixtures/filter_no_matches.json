{
  "type_name": "MassiveType",
  "string_filter": "nothing"
}
