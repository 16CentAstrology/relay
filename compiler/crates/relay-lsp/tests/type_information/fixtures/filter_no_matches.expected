==================================== INPUT ====================================
{
  "type_name": "MassiveType",
  "string_filter": "nothing"
}
==================================== OUTPUT ===================================
type MassiveType {
  # 1000 fields not matching filter
}
